src/compiler/commandLineParser.ts

```diff
@@ -426,6 +434,7 @@ namespace ts {
             // use type = object to copy the value as-is
             name: "paths",
             type: "object",
+            affectsModuleResolution: true,

```

Comment:
While you have added this, could you please change ```changesAffectModuleResolution``` as well to use this instead of manual checks.

Reply:
:+1: